src/pages/about.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import styled from "styled-components";
+import AboutDetail from "../components/About/AboutDetail";
+import AboutInfo from "../components/About/AboutInfo";
+import { Container } from "@components/styled";
+
+function About() {
+  const [currentSelected, setcurrentSelected] = useState<string>("overview");
+
+  return (
+    <Container bg="#FAFAFA" paddingMultiplier={0}>
+      <StyledAbout>
+        <AboutInfo setCurrentSelected={setcurrentSelected} />
```

Comment:
Your implementation of the tabs works and it's beautiful. From a scalability POV, consider using this datastructure to define your data `
[
    {
        title:"Company overview",
        content: then your content here
    },
    {
        title:"Company overview",
        content: then your content here
    },
    {
        title:"Company overview",
        content: then your content here
    }
]`